routes/index.js

```diff
@@ -20,6 +20,44 @@ function buildMessage(req) {
   }
 }
 
+function buildStoriesMessage(req) {
+  return {
```

Comment:
Could utilize DRY here - maybe create a function that combines this with the default buildMessage, or something?